packages/sources/svelte/src/components/navigation/VtmnSearch/VtmnSearch.svelte

```diff
@@ -0,0 +1,109 @@
+<script>
+  import { cn } from '../../../utils/classnames';
+
+  import { VtmnButton } from '@vtmn/svelte';
+  import { createEventDispatcher } from 'svelte';
+  import { VtmnSearchVariant, VtmnSearchSize } from './enums';
```

Comment:
```suggestion
  import { VARIANTS, SIZES } from './enums';
```

Reply:
See with Laurent, the new naming is `VTMN_{component name}_{enum name}` -> `VTMN_SEARCH_SIZES`